### ISSUE TITLE
problem with resolution 1 if resolution 0 not added

### ISSUE THREAD

**Issue by jkosinski** [salilab/pmi#136]
I think, when creating a system with PMI, selecting by resolution=1 doesn't work properly if you do not add resolution 0:

If in:
test_topology.TopologyTest.test_build_system
you replace:

``` python
m1.add_representation(atomic_res,resolutions=[0,1,10])
```

with

``` python
m1.add_representation(atomic_res,resolutions=[1,10])
```

and comment out testing resolution 0, the             

``` python
self.assertEquals(IMP.atom.Residue(res1[0]).get_residue_type(),
                              topology.get_residue_type_from_one_letter_code(rname))
```

will fail because res1 is now ["Fragment [1-3)"] instead of ["Residue_1"]

Basically, it selects the resolution 10 instead of 1.

**Comment by cgreenberg** [comment]
Yes, I have reproduced the problem. Strangely only happens if you have res=[1,10], not if [1] only

**Comment by cgreenberg** [comment]
Ah, I recall the bug is due to the creation of the dummy chain for averaging along backbone.

**Comment by crosslinks** [comment]
great you know what's going. For me is pretty crucial this one - I wanted speed simulations up by attaching excluded volume restraint and em restraint to lower resolution, xlink restraint to higher res.

**Comment by cgreenberg** [comment]
Not so simple...looks like the bug is actually in the Representation decorator itself....

**Comment by jkosinski** [comment]
So do you mean this is a problem with IMP rather than PMI? Do you or Ben think this could be solved anytime soon?

**Comment by cgreenberg** [comment]
Finally fixed this with salilab/imp@d4609a51544562b8427de0eb8666269049ae503a
Also added a test to PMI.
